packages/synthetix-main/contracts/mocks/token/CollateralMock.sol

```diff
@@ -23,12 +24,24 @@ contract CollateralMock is ERC20, IRewardDistributor {
 
     // permissionless payout for convenience testing
     function payout(
+        uint128,
         uint128,
         address,
-        address to,
+        address sender,
         uint amount
     ) external returns (bool) {
-        _mint(to, amount);
+        _mint(sender, amount);
```

Comment:
Note that this function should revert if the caller isn't the main proxy, maybe leave a note with a comment here:

`// IMPORTANT: In production, this function should revert if msg.sender is not the Synthetix CoreProxy address.`